docs/guide/three/glitch.md

```diff
@@ -0,0 +1,34 @@
+# Glitch
+
+<DocsDemo>
+  <GlitchThreeDemo />
+</DocsDemo>
+
+Glitch is an effect that simulates that simulates digital glitches, which are random or intentional errors that occur in digital media such as images, videos, or audio. The effect can be used to create a distorted or disrupted appearance, with elements of the scene appearing to shift or flicker in a chaotic manner.
+
+The native `<Glitch />` pass in TresJS allows you to add this effect to your 3D scenes, and provides several parameters that can be tweaked to achieve the desired glitchy look, such as delay, duration, strength, mode, and more. The end result can be a unique and striking visual style that adds an extra layer of interest to your 3D scenes.
```

Comment:
```suggestion
The native `<Glitch />` component allows you to add this effect to your 3D scenes, and provides several parameters that can be tweaked to achieve the desired glitchy look, such as delay, duration, strength, mode, and more. The end result can be a unique and striking visual style that adds an extra layer of interest to your 3D scenes.
```

This paragraph could be omitted, since all the infos in there are already present on the page or quite obvious.